docs/guides/visualize-circuit-timing.ipynb

```diff
@@ -0,0 +1,7874 @@
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from qiskit import QuantumCircuit\n",
+    "from qiskit.visualization.timeline import draw, IQXDebugging\n",
+    "from qiskit.providers.fake_provider import GenericBackendV2\n",
+    "from qiskit.transpiler import generate_preset_pass_manager\n",
+    "\n",
+    "qc = QuantumCircuit(2)\n",
+    "qc.h(0)\n",
+    "qc.cx(0, 1)\n",
+    "qc.measure_all()\n",
+    "\n",
+    "backend = GenericBackendV2(5)\n",
+    "pm = generate_preset_pass_manager(backend=backend, optimization_level=1)\n",
+    "isa_circuit = pm.run(qc)\n",
+    "draw(isa_circuit, style=IQXDebugging(), target=backend.target)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5406fe18",
+   "metadata": {},
+   "source": [
+    "You can create custom generator or layout functions and update an existing stylesheet with the custom functions. This way, you can control the most of the appearance of the output image without modifying the codebase of the scheduled circuit drawer.  See the [`timeline_drawer`](/docs/api/qiskit/qiskit.visualization.timeline_drawer) API reference for more examples."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ad24e3a0",
+   "metadata": {},
+   "source": [
+    "<span id=\"qr-support\"></span>\n",
+    "## Qiskit Runtime support\n",
+    "\n",
+    "While the timeline drawer built in to Qiskit is useful for static circuits, it might not accurately reflect the timing of [dynamic circuits](/docs/guides/classical-feedforward-and-control-flow) because of implicit operations such as broadcasting and branch determination. As part of dynamic circuit support, Qiskit Runtime returns the accurate circuit timing information inside the job results when requested.\n",
+    "\n",
+    "<Admonition type=\"note\" title=\"Notes\">\n",
+    "- This is an experimental function. It is in preview release status and is therefore subject to change.\n",
+    "- This function only applies to Qiskit Runtime Sampler jobs.\n",
+    "</Admonition>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "995a99e8",
+   "metadata": {},
+   "source": [
+    "### Enable timing data retrieval\n",
+    "\n",
+    "To enable timing data retrieval, set the experimental `scheduler_timing` flag to `True` when running the primitive job."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "ab8c6c23",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "management.get:WARNING:2025-10-20 12:31:57,331: Loading default saved account\n",
+      "qiskit_runtime_service.__init__:WARNING:2025-10-20 12:32:05,740: Instance was not set at service instantiation. Free and trial plan instances will be prioritized. If you need a specific instance set it explicitly either by using a saved account with a saved default instance or passing it in directly to QiskitRuntimeService().\n"
```

Comment:
I don't think we want to print this

Reply:
> I don't think we want to print this

I agree, I asked Eric if there's any way to hide the output, but I'm not sure there is.